gdal/scripts/setdevenv.sh

```diff
@@ -14,26 +14,26 @@ if [[ $called == "$0" ]]; then
     exit 1
 fi
 
-GDAL_ROOT=$(cd $(dirname ${BASH_SOURCE[0]})/..; pwd)
+GDAL_ROOT=$(realpath -e $(dirname ${BASH_SOURCE[0]})/..)
```

Comment:
Apparently realpath is not available on Mac: https://github.com/google/enjarify/issues/9

Reply:
OK, will use `readlink(1)` instead.